src/main/java/seedu/address/logic/commands/AddCompanyCommand.java

```diff
@@ -28,7 +28,8 @@ public class AddCompanyCommand extends AddCommand<Company> {
             + PREFIX_ADDRESS + "12 Ang Mo Kio Street 64, Singapore 569088 "
             + PREFIX_BILLING_DATE + "5 "
             + PREFIX_PHONE + "64815511";
-    public static final String MESSAGE_SUCCESS = "New company added: %1$s";
+    public static final String MESSAGE_SUCCESS = "Company added: %1$s; Address: %2$s; "
+            + "Billing date: %3$s; Phone: %4$s;";
```

Comment:
Just to confirm, so the desired format is as follows?

<img width="987" alt="image" src="https://github.com/user-attachments/assets/897be8e0-7f13-4394-8e86-c0f47b2a37dd">

I'm okay with it, but don't merge this PR yet till Alek takes a look as it relates to him.

Reply:
However, do note that on the default window size, text for even yours would be partially covered.

<img width="955" alt="image" src="https://github.com/user-attachments/assets/8ae76d11-4cb2-4f52-9e3e-76759d4e0746">

Reply:
I'm not sure how bad it is to overflow like that since the scrollbar is present. The invalid command format also causes text to be displayed out of the visible box.